packages/connect/src/utils/debug.ts

```diff
@@ -159,3 +159,36 @@ export const getLog = () => {
     logs.sort((a, b) => a.timestamp - b.timestamp);
     return logs;
 };
+
+let logWorker: SharedWorker | undefined;
```

Comment:
not sure about this line. Lets say, my project is configured for node. There is no SharedWorker. It could cause troubles. Can't we have something more abstract here?

Reply:
Sure, I moved the whole new utils for shared worker to `iframe` to packages/connect-iframe/src/sharedLoggerUtils.ts

It makes more sense since sharedlogger is there as well and we do not want to pollute core with web stuff.